src/random_test.cpp

```diff
@@ -1361,6 +1386,13 @@ void create_in_memory_data() {
     g_row_format.clear();
     g_key_block_size.clear();
   }
+
+  int undo_tbs_count = opt_int(NUMBER_OF_UNDO_TABLESPACE);
+  if (undo_tbs_count>0) {
```

Comment:
Fix the indentation in . its "if (a > b) not if(a>b)"
we are using clang-format version 8.0.0 for formating.
you can integrate that with vim.  

please fix it before push.

Reply:
Fixed